README.md

```diff
@@ -111,7 +143,9 @@ Then, in your `.eslintrc.json`:
 
 ---
 
-This plugin works best if you disable all other ESLint rules relating to code formatting, and only enable rules that detect patterns in the AST. (If another active ESLint rule disagrees with `prettier` about how code should be formatted, it will be impossible to avoid lint errors.) You can use [eslint-config-prettier](https://github.com/prettier/eslint-config-prettier) to disable all formatting-related ESLint rules. If your desired formatting does not match the `prettier` output, you should use a different tool such as [prettier-eslint](https://github.com/prettier/prettier-eslint) instead.
+## Note
+
+If your desired formatting does not match the `prettier` output, you should use a different tool such as [prettier-eslint](https://github.com/prettier/prettier-eslint) instead.
```

Comment:
Nitpick: I think it would be better to keep this note next to the comments about `eslint-config-prettier`. Can you move it to the `Recommended Configuration` section (maybe under the first paragraph)?

Reply:
Done.